pytorch_cnn_trainer/engine.py

```diff
@@ -72,6 +72,9 @@ def train_step(
     top1_m = utils.AverageMeter()
     top5_m = utils.AverageMeter()
     cnt = 0
+    losses_t = []
```

Comment:
May I know how this solves the problem?
Why would you collect every `train_step` metrics?
Train_step was returning them anyways can we use that directly to `fit()` ?

Reply:
actually I did'nt try printing train_metrics  so I thought it was not storing values of metrics but just updating a variable and value for previous epoch would be lost so I used list to store each

Reply:
It actually stores metrics in oredered dict.